pytato/array.py

```diff
@@ -2180,4 +2192,30 @@ def amin(a: Array, axis: Optional[Union[int, Tuple[int]]] = None) -> Array:
 
 # }}}
 
+
+# {{{ Communication nodes
+
+
+class DistributedSend(Array):
```

Comment:
```suggestion
class DistributedSend(DictofNamedArrays):
```

Reply:
I thought `Array` for this is fine, no?

Reply:
Discussing with @inducer, we thought that since DistributedSend doesn't really return anything, a DictOfNamedArrays might be more appropriate.

Reply:
But we do pack it into AdditionalOutput, and I think there's going to be a one-to-one mapping from a send-to-receive, unless I am mistaken.

Reply:
> one-to-one mapping from a send-to-receive, unless I am mistaken.

Not necessarily, IMO.

Imagine (upwided) FD 1D advection: it has some sends that have no corresponding receives, and vice versa.

Reply:
> But we do pack it into AdditionalOutput

We could change that to take a `DictOfNamedArray`s.